### ISSUE TITLE
automatically navigate user view to main view after tapping `Water Now`

### ISSUE THREAD

**Issue by foss-** [abertschi/water-me#13]
**Currently**

When tapping `Water Now` the info `watered x days ago` is updated. But to move to the next plant, user have to tap back arrow first.

**Expected**

There is nothing much to do after tapping `Water Now` for a specific plant entry. It would be convenient to automatically navigate users to main view after that action so that they can continue with the next plant entry.

Bonus points for not using all caps for the `Water Now` button. There's no need to scream at the user at that point in time :)

**Comment by abertschi** [comment]
Thanks for reaching out and using the app :+1: 

> There is nothing much to do after tapping Water Now for a specific plant entry. It would be convenient to automatically navigate users to main view after that action so that they can continue with the next plant entry.

For watering multiple plants, there is the feature to swipe the plant on the main screen. Swipe an entry to either right or left and a text appears to 'Mark as watered'.

> Bonus points for not using all caps for the Water Now button. There's no need to scream at the user at that point in time :)

Currently, all actions are upper case, including the undo buttons of the Snackbar. I guess this boils down to preference and design style.

There is a list of feature-request to implement in case you want to contribute to the project. See other issues in the repo.
Cheers

**Comment by foss-** [comment]
Swiping makes sense, however it is hard to discover. Noticed when swiping LTR  info `Mark as watered` shows however swiping RTL is just blank.

**Comment by abertschi** [comment]
do you have a suggestion to improve it? custom actions on swipe in a list is a common design pattern found in many applications. It is also featured in the screenshot in the README and the f-droid store.

thanks and cheers.

**Comment by foss-** [comment]
Good point, I had not checked screenshots and sure swiping on lists isn't an exotic event. All is good, I think we can even close this issue if you want.  I would still prefer the behavior on Water now tap to go back to main view and to also have the info on RTL swipe what the swipe will do.

But other than that my initial complaint is well covered with the swipe gesture.

**Comment by abertschi** [comment]
No we can leave this open. These are reasonable requests:
- jump back to plant list after marking a plant as watered in plant detail screen
- add label mark as watered on left and right swipe in plant list. 

Cheers
